website_and_docs/content/documentation/test_practices/design_strategies.ja.md

```diff
@@ -0,0 +1,370 @@
+---
+title: "デザインパターンと開発戦略"
+linkTitle: "デザイン戦略"
+weight: 1
+---
+(以前の場所: https://github.com/SeleniumHQ/selenium/wiki/Bot-Style-Tests)
+
+## 概要
+時間の経過とともに、プロジェクトは多数のテストが積み上がる傾向があります。 
+テストの総数が増えると、コードベースに変更を加えることが難しくなります。
+アプリケーションが正常に機能していても、1回の"単純な"変更で多数のテストが失敗する可能性があります。 
+これらの問題が避けられない場合もありますが、問題が発生した場合は、できるだけ早く稼働を再開する必要があります。 
+次のデザインパターンと戦略は、テストの作成と保守を容易にするためにWebDriverで以前に使用されています。 
+それらもあなたにとって役に立つかもしれません。
+
+[DomainDrivenDesign](encouraged/domain_specific_language.ja.md)：アプリのエンドユーザーの言語でテストを表現します。   
+[PageObjects](encouraged/page_object_models.ja.md)：WebアプリのUIの単純な抽象化  
```

Comment:
[PageObjects]({{< ref "//encouraged/page_object_models.ja.md" >}})

Reply:
This should fix the broken links. I have updated it for en, zh-cn, pt-br docs!

Reply:
@harsha509 Thank you. I've fixed these broken links.